### DISCUSSION TITLE
how to create multiple table in same sheet ?

### DISCUSSION THREAD

**Discussion by bushan1891** [exceljs/exceljs#2512] [Q&A]
This is not an issue ! is it possible to create multiple tables on same sheet  ?

I tried redefining the column but it edits the same table 

any help on this is appreciated !

**Comment by jpadilla** [comment]
Wondering about this same thing.

**Comment by mmattosr** [comment]
You can create multiples sheets into a workbook, maybe this solve your problem

**Comment by jtraband** [comment]
same issue, don't really want to create another sheet.

**Comment by manishwaran** [comment]
Any update regarding this ?

**Comment by bushan1891** [comment]
@all there are ways to do this not sure if it the best practice 

https://github.com/bushan1891/custom-ServerSide/blob/master/controllers/util/xlsx_generator_util.js

Hope this helps

**Comment by ysfaran** [comment]
Is this still not possible?

Whenever I add a second table using `worksheet.addTable` the first table might be printed but only the second table is actually an excel table.

IMHO I think it's pretty normal to have more than one table in the same excel sheet. I'm really confused by this behaviour..

**Comment by ysfaran** [comment]
It's actually possible to add multiple tables to the same sheet. You need to take care that all your tables have a different `name`. Even accross sheets the table names need to be unique:

```ts
worksheet.addTable({
  name: "MyUniqueTableName",
  // ...
});
```

I think this ticket should be closed.
